modelplace_api/visualization.py

```diff
@@ -67,6 +67,39 @@ def draw_line(
             i += 1
 
 
+def draw_text_label(
+    image: np.ndarray,
+    detection: Union[AgeGenderLabel, EmotionLabel],
+    text: str,
+    font_scale: float = 0.6,
+    thickness: int = 1,
+    bg_color: tuple = RGBA_COLORS[196][2::-1],
+    text_color: tuple = (255, 255, 255),
```

Comment:
minor: @PVSemk probably we should use color from RGBA_COLORS as you said earlier